app/code/Magento/CatalogInventory/Block/Plugin/ProductView.php

```diff
@@ -1,26 +1,26 @@
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Copyright 2015 Adobe
+ * All Rights Reserved.
  */
 namespace Magento\CatalogInventory\Block\Plugin;
 
-use Magento\CatalogInventory\Api\StockRegistryInterface;
+use Magento\CatalogInventory\Model\Product\QuantityValidator;
 
 class ProductView
 {
     /**
-     * @var StockRegistryInterface
+     * @var QuantityValidator
      */
-    private $stockRegistry;
+    private $productQuantityValidator;
 
     /**
-     * @param StockRegistryInterface $stockRegistry
+     * @param QuantityValidator $productQuantityValidator
      */
     public function __construct(
-        StockRegistryInterface $stockRegistry
+        QuantityValidator $productQuantityValidator
```

Comment:
Use ObjectManager to initialize as this change is BiC.

Reply:
done